test/decorator.test.js

```diff
@@ -660,3 +660,53 @@ test('a decorator should addSchema to all the encapsulated tree', t => {
 
   fastify.ready(t.error)
 })
+
+test('after can access to a decorated instance and previous plugin decoration', t => {
+  t.plan(11)
+  const TEST_VALUE = {}
+  const OTHER_TEST_VALUE = {}
+  const NEW_TEST_VALUE = {}
+
+  const fastify = Fastify()
+
+  fastify.register(fp(function (instance, options, next) {
+    instance.decorate('test', TEST_VALUE)
+
+    next()
+  }))
+    .after(function (err, instance, done) {
```

Comment:
can you move this on the above line? Like:

```js
  })).after(function (err, instance, done) {
```

Reply:
done